ckanext/canada/cli.py

```diff
@@ -683,25 +713,45 @@ def _load_suggested(use_created_date, filename):
             "state": "active",
             "id": uuid,
             "title_translated": {
-                'en': str(row['title_en'], 'utf-8'),
-                'fr': str(row['title_fr'], 'utf-8')
+                # type_ignore_reason: checking existance
+                'en': str(row['title_en'],  # type: ignore
+                          'utf-8') if row['title_en'] else '',
```

Comment:
```suggestion
                'en': row['title_en'],
```
Should work fine now that `.encode('utf-8')` has been removed above?

For the type checks we could define the set of keys with a `class Suggested(TypedDict): ...` and then cast the row? There must be a clean way of doing DictReader dict key checks with pyright.

Reply:
Yeah, I personally have never used this command. Nor had to do anything with suggested datasets to this date. So didn't really put much effort into this one.

I have cleaned up this encoding stuff now. We are on Python3 now so hopefully it is all just fine. We will find out if we ever actually have to use this command I guess.